test/jdk/java/awt/Focus/UnaccessibleChoice/AccessibleChoiceTest.java

```diff
@@ -42,21 +41,21 @@
 
 import javax.imageio.ImageIO;
 
-/**
+/*
  * @test
  * @bug 4478780
  * @key headful
  * @summary Tests that Choice can be accessed and controlled by keyboard.
  */
+
 public class AccessibleChoiceTest {
     //Declare things used in the test, like buttons and labels here
-    Frame frame = new Frame("window owner");
-    Window win = new Window(frame);
+    Frame frame = new Frame("Accessible Choice Test Frame");
     Choice choice = new Choice();
     Button def = new Button("default owner");
     CountDownLatch go = new CountDownLatch(1);
 
-    public static void main(final String[] args) throws IOException {
+    public static void main(final String[] args) throws Exception {
         AccessibleChoiceTest app = new AccessibleChoiceTest();
         app.test();
```

Comment:
Since you are doing a bit of modifictaion to the test., you may think of modifying it further.
1. Don't think you need to create the object.
2. UI creation code can be moved to more meaningful method name `createAndShowUI instead of init` inside EDT block and followed by call to `test` method.

Reply:
Sure. It cleans the test up. Wasn't in my initial plan, but it's a good point. Done.